adders/tailwindcss/config/adder.ts

```diff
@@ -92,7 +92,10 @@ export const adder = defineAdderConfig({
             content: ({ js, html }) => {
                 js.imports.addEmpty(js.ast, "../app.css");
                 const slot = html.element("slot");
-                html.ast.childNodes.push(slot);
+                // @ts-expect-error should be compatible
+                if (!js.common.hasNode(html.ast.childNodes, slot)) {
```

Comment:
why should we have to check if there's a slot at all?  I don't quite understand why we need to do anything more than add the stylesheet for a css integration

Reply:
> why should we have to check if there's a slot at all?

it covers the case of if the `+layout.svelte` isn't already present in the project, then the adder will create the layout file and add the `slot` element

having a second look now though, I was definitely overthinking this. If the `+layout.svelte` file is newly created, then `html.ast.childNodes` should be empty, so we can just check for that instead